### PR TITLE
Fix bug #98 and improve code readability

### PR DESCRIPTION
**Pull request by jiangliu**
1. Refine code for maintenance.
2. Fix bug #98

### CODE REVIEW

File: src/agent/src/linux_abi.rs [hunk 1]
```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2019 Ant Financial
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Linux ABI related constants.
+
+pub const SYSFS_DIR: &str = "/sys";
+
+pub const SYSFS_PCI_BUS_PREFIX: &str = "/sys/bus/pci/devices";
```
Comment (jodh-intel) [inline src/agent/src/linux_abi.rs]: It's really annoying that constant string cannot be joined at compile time in rust, but adding https://github.com/Vurich/const-concat seems overkill here.

File: src/agent/src/linux_abi.rs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+pub const SYSFS_PCI_BUS_RESCAN_FILE: &str = "/sys/bus/pci/rescan";
+#[cfg(any(
+    target_arch = "x86_64",
+    target_arch = "x86",
```
Comment (jodh-intel) [inline src/agent/src/linux_abi.rs]: Nit: would be good if this list was sorted alphabetically.

File: src/agent/src/uevent.rs [hunk 3]
```diff
@@ -28,36 +22,105 @@ pub struct Uevent {
...
+                if let Err(e) = online_device(online_path.as_ref()) {
+                    error!(
+                        *logger,
+                        "failed to online device";
+                        "device" => &self.devpath,
+                        "error" => format!("{}", e),
+                    );
+                }
+            }
+        }
```
Comment (jodh-intel) [inline src/agent/src/uevent.rs]: I wonder if we want to add an:

```rust
else {
    debug!(*logger, "ignoring event"; "uevent" => format!("{:?}", self));
}

**Comment by yyyeerbo** [comment]
build fails. maybe missing some use xxx::CGROUPPATH..

File: src/agent/src/linux_abi.rs (hunk 2 above)
Comment (jiangliu) [reply src/agent/src/linux_abi.rs]: fixed.

File: src/agent/src/uevent.rs (hunk 3 above)
Comment (jiangliu) [reply src/agent/src/uevent.rs]: Done.

**Comment by jiangliu** [comment]
> build fails. maybe missing some use xxx::CGROUPPATH..

Fixed:)

**Comment by jodh-intel** [comment]
Thanks @jiangliu !

Ping @kata-containers/agent.

**Pull request merged**